webviz_subsurface/_utils/simulation_timeseries.py

```diff
@@ -118,15 +116,13 @@ def add_fanchart_traces(ens_stat_df, vector, color, legend_group: str, line_shap
         },
         {
             "name": legend_group,
-            "hovertext": "Minimum",
+            "hovertext": "Mean",
             "x": ens_stat_df[("", "DATE")],
-            "y": ens_stat_df[(vector, "min")],
+            "y": ens_stat_df[(vector, "mean")],
             "mode": "lines",
-            "fill": "tonexty",
-            "fillcolor": fill_color,
-            "line": {"width": 0, "color": line_color, "shape": line_shape},
+            "line": {"color": line_color, "shape": line_shape},
             "legendgroup": legend_group,
-            "showlegend": False,
+            "showlegend": True,
```

Comment:
A bit strange, but had to alter the traces to get the right fading of the fanchart. 
On the other hand: the new order fits better with the Plotly reference guide and how fill is expected to behave.

Reply:
Turned out that it is because the previous one used "regular percentiles", but here I have switched P10 and P90 to the oil industry convention (p90 low, p10 high)